client/src/components/GameInfoCarousel.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState, useEffect, useReducer } from 'react';
+import axios from 'axios';
+import { MainGameInfoWrapper, BackGroundWaterMark } from './styles';
+import ImageCarousel from './ImageCarousel';
+
+// const gameReducer = (state, action) => {
+//   switch (action.type) {
+//     case 'MAIN_IMAGE':
+//       return action.image;
+//     default:
+//       return state;
+//   }
+// };
+// dispatch({ type: 'MAIN_IMAGE', image: res.data[0].game_photo })
+
+const GameInfoCarousel = () => {
+  const [currentGameId] = useState(7);
+  const [mainImage, setMainImage] = useState('');
+  // const [image, dispatch] = useReducer(gameReducer, '');
+  // const [mainImage, setMainImage] = useState('');
+  const [images, setImageCarousel] = useState([]);
+
+  useEffect(() => {
```

Comment:
I'm still a little unfamiliar with useEffect and  useState - seems to be hooks related, overall looks pretty clean!